src/js/controller/controller.js

```diff
@@ -573,6 +577,7 @@ Object.assign(Controller.prototype, {
             const adState = _getAdState();
 
             if (adState) {
+                _this._instreamAdapter.noResume = false;
```

Comment:
Added this to address concern about not being able to `resume` after unpausing.

Reply:
I was hoping to see a resolution to this issue that replaced code rather than just adding more code on top of what already does not work.

Reply:
I'll give this some more thoughts and re-open. Thanks!